crypto/s2n_hmac.c

```diff
@@ -26,6 +26,9 @@
 #include "utils/s2n_blob.h"
 #include "utils/s2n_mem.h"
 
+#include <inttypes.h>
```

Comment:
I haven't seen `inttypes.h` before, but from looking at https://stackoverflow.com/questions/7597025 it seems that it's a superset of `stdint.h` that also includes support for `printf` functions.

It doesn't look like we're using those `printf` functions though, so I think we should use `stdint.h` instead.

Reply:
Done